modules/schema/src/main/java/org/apache/ignite/internal/schema/configuration/TableValidatorImpl.java

```diff
@@ -40,9 +42,19 @@ public class TableValidatorImpl implements Validator<TableValidator, NamedListVi
     /** {@inheritDoc} */
     @Override
     public void validate(TableValidator annotation, ValidationContext<NamedListView<TableView>> ctx) {
+        TablesView tablesConfig = ctx.getNewRoot(TablesConfiguration.KEY);
+
+        Set<String> idxNames = tablesConfig == null
+                ? Collections.emptySet()
+                : tablesConfig.indexes().namedListKeys().stream().map(String::toUpperCase).collect(Collectors.toSet());
```

Comment:
Why do you convert names to upper case?

Reply:
Fixed, the case-insensitive comparison has been removed.